modules/data.atmosphere/R/check_met_input.R

```diff
@@ -94,7 +94,7 @@ check_met_input_file <- function(metfile,
     test_raw = purrr::map(nc_vars, check_unit, nc = nc, variable_table = variable_table),
     test_passed = !purrr::map_lgl(test_raw, inherits, "try-error"),
     test_error_message = purrr::map_chr(test_raw, purrr::possibly(as.character, NA_character_))
```

Comment:
```suggestion
    test_passed = !purrr::map_lgl(.data$test_raw, inherits, "try-error"),
    test_error_message = purrr::map_chr(.data$test_raw, purrr::possibly(as.character, NA_character_))
```

TIL you can reuse previously defined columns inside a tibble definition! I had assumed this kind of transformation would need to be written `df <- tibble::tibble(col_a = ...) %>% mutate(col_b = f(col_a, ...),...)`

Reply:
Yup! IIRC, this is one of the key differences between `tibble::tibble` and `data.frame` -- it will _not_ work in the latter.